### PR TITLE
DOTNET_SKIP_FIRST_TIME_EXPERIENCE

### PR DESCRIPTION
**Pull request by sungam3r**
To get rid of
изображение

### CODE REVIEW

File: .github/workflows/wipcheck.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     types: 
       - opened        # when PR is opened
-      - edited        # when PR title is edited
+      - edited        # when PR is edited
```
Comment (sungam3r) [inline .github/workflows/wipcheck.yml:7]: Not only title. I noticed that editing my comment also triggers wipcheck.

**Comment by sungam3r** [comment]
Also see this https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet#environment-variables

File: .github/workflows/wipcheck.yml (hunk 1 above)
Comment (sungam3r) [reply .github/workflows/wipcheck.yml:7]: Well, it is true only for the first comment. Maybe it is treated in some special way.

File: src/Tests.props [hunk 2]
```diff
@@ -6,7 +6,7 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
-    <PackageReference Include="Shouldly" Version="4.0.0-beta0002" />
+    <PackageReference Include="Shouldly" Version="4.0.0-beta0004" />
```
Comment (sungam3r) [inline src/Tests.props:9]: I changed this to trigger actions on `src`

**Review by Shane32 — APPROVED** [review]
I really don’t understand why you’d want to reduce the log output of a workflow. GitHub collapses the information anyway so you don’t see it unless you expand the details for that action.  So I wouldn’t but it doesn’t really matter so ok.....

**Comment by sungam3r** [comment]
Yes, you are right, there is not much difference, I just wanted to try it for a long time.

**Pull request merged**